mssqlcli/main.py

```diff
@@ -411,6 +444,14 @@ def run_cli(self):
                 if quit_command(document.text):
                     raise EOFError
 
+                try:
+                    document = self.handle_editor_command(self.cli, document)

```

Comment:
Should you be passing in the document.text rather than the document object. Looking at handle_editor_command it doesn't seem like it uses the document instance other than for it's text. For readability and uniformity I think you could try the signature like the quit_command method.

Reply:
Never mind, I see how it is used. It makes more sense to pass in the document. Disregard my previous comment.